charts/seed-bootstrap/charts/fluent-bit/templates/fluent-bit-priorityclass.yaml

```diff
@@ -1,7 +0,0 @@
-apiVersion: {{ include "priorityclassversion" . }}
```

Comment:
We should pro-actively delete this `PriorityClass` at the end of the seed reconciliation

Reply:
Should't we just delete it once as we did [here](https://github.com/gardener/gardener/blob/0745524969ed3da045820a360213f648e84440c7/pkg/gardenlet/controller/factory.go#L142) instead of trying to delete it in the reconcile loop? Also I guess this applies for the other priority classes that are dropped as well.

Reply:
It doesn't matter much, I think. In the case of `gardenlet`'s PC, the difference is that when this code is reached, gardenlet is already running with a new PC and can safely delete its old one.
With the other PCs, if we delete them already in `factory.go`, the system components might fail if they get restarted by VPA and the PC is missing (in case the seed reconciliation fails for any reason before it creates the new PCs).